### DISCUSSION TITLE
epoch times please

### DISCUSSION THREAD

**Discussion by rob0101** [amberelectric/public-api#9] [Ideas]
Rather than everyone needing to do timezone gymnastics, please put the unix / epoch time in the responses.

"nemTime": "2021-06-21T13:30:00+10:00",
"nemTime_epoch" : 1624245845,

Every programming language can convert from epoch times to any timezone the user wants.  Going back is much harder and should be done once by the server.

**Comment by madpilot** [comment]
Use `startTime` and `endTime` if you want UTC time. `nemTime` has special meaning, which is why it's in the +10 timezone.

See the docs.

**Reply by madpilot** [reply]
Every language can also parse ISO date times.

**Reply by rob0101** [reply]
I'm not asking to change any of the timestamps you have,  only to add the epoch time.

   **"Every language can also parse ISO date times."**
I assumed that too until I tried to import AEMO type data into FileMaker Pro a couple of weeks ago.

**Reply by madpilot** [reply]
Ah. Right. Leave that with me...
